cfgov/unprocessed/css/molecules/translation-links.less

```diff
@@ -58,3 +63,36 @@ html[lang='ar'] ul.m-translation-links {
     }
   }
 }
+
+//fix issue with padding/margin and verticle rule appearing to the left of print icon
+html[lang='ar'] .m-social-media {
```

Comment:
We'll want to place the RTL code for `m-social-media` in the same location as the LTR `m-social-media` code, namely here https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/unprocessed/css/molecules/social-media.less#L94

Reply:
All "social-media" css has been moved to social-media.less